app/models/place_bid.rb

```diff
@@ -65,7 +60,7 @@ def validate_bid_data
       fail UnauthorizedError, 'Bids must be in increments of one dollar'
     end
 
-    if amount > BID_LIMIT
+    if amount > auction.start_price
```

Comment:
We got lucky since no auctions started at a number different than 3,500, but yay!